packages/chart/src/scss/main.scss

```diff
@@ -247,6 +247,47 @@
         }
       }
     }
+    .legend-group {
```

Comment:
Since this is part of Legend.Subgroup.tsx maybe the styles should be imported from their own file that sits next to it. LegendSubroup.styles.css.